docs/gitbook/tips/mixserver.md

```diff
@@ -32,8 +33,8 @@ Running Mix Server
 ===================
 
 First, put the following files on server(s) that are accessible from Hadoop worker nodes:
-* [target/hivemall-mixserv.jar](https://github.com/apache/incubator-hivemall/releases)
-* [bin/run_mixserv.sh](https://github.com/apache/incubator-hivemall/raw/master/bin/run_mixserv.sh)
+* [target/hivemall-mixserv.jar](https://github.com/myui/hivemall/releases)
+* [bin/run_mixserv.sh](https://github.com/myui/hivemall/raw/master/bin/run_mixserv.sh)
```

Comment:
@myui It seems there are no `hivemall-mixserv.jar` and `bin/run_mixserv.sh` in current repo. It'd be nice you to suggest alternative link. Or, I will keep current description.